### ISSUE TITLE
Does tufte-css play nicely with screen readers?

### ISSUE THREAD

**Issue by RichMorin** [edwardtufte/tufte-css#183]
As a sighted person, and a long-time fan of ET's work, I applaud this project. However, I also realize that many things that it suggests (e.g., embedded graphics) would be difficult or even impossible to make accessible for blind, dyslexic, and/or visually impaired readers.

That said, I wonder whether the remaining features and practices (e.g., document structure) have been evaluated to determine whether they play nicely with screen readers.  If not, perhaps they should. I'm not able to make this evaluation myself, but some folks on email lists I frequent might be. I'd be happy to help the project make contact with them, relay their reactions, etc.

P.S. The [tufte-css](https://edwardtufte.github.io/tufte-css) page is _very_ nicely done, exemplifying that which it describes. Indeed, it reminds me of [Ars Poetica](https://www.poetryfoundation.org/poetrymagazine/poems/17168/ars-poetica).

**Comment by foxihd** [comment]
Hi, for what I understand, CSS doesn't work fully with the assistive technology. Additional WAI-ARIA attribute and proper semantic HTML structure is required to satisfy the screen readers.

If you are interested, I write a similar project using Hugo processor, I'd be honored if you'd like to check out my project; [hugo-et-hd](https://github.com/foxihd/hugo-et-hd/).
Hugo ET-HD is Tufte CSS derivative, even more printer friendly, accessibility-enhanced Hugo theme.
There's also legibility control for Bionic-Reading-like, OpenDyslexic typeface override, font & baseline scaling, color scheme & contrast control. It's far to be well rounded, but "it works".
Any feedback will be much appreciated.
